remediation/self-node-remediation/configuration.md

```diff
@@ -10,8 +10,17 @@ nav_order: 2
 Self Node Remediation utilizes a custom resource (CR) to define a custom configuration.
 The `SelfNodeRemediationConfig` CR is automatically created upon installation.
 Currently there are two configuration fields:
-1. Watchdog device - the file path of the watchdog in the nodes. If this path doesn't exist, Self Node Remediation will use a software reboot.
-2. Safe timeout - Self Node Remediation needs to assume the timeout that a unhealthy node has been rebooted. Self Node Remediation automatically calculates the lower bound for this value, but it can and should be changed to an upper value, in case of non-homogeneous hardware where different nodes have different watchdog timeout.
+1. safeTimeToAssumeNodeRebootedSeconds: 180 - Specify the timeout duration for the surviving peer, after which the Operator can assume that an unhealthy node has been rebooted. The Operator automatically calculates the lower limit for this value. However, if different nodes have different watchdog timeouts, you must change this value to a higher value.
+2. watchdogFilePath: /dev/watchdog - Specify the file path of the watchdog device in the nodes. If you enter an incorrect path to the watchdog device, the Self Node Remediation Operator automatically detects the softdog device path.
+   If a watchdog device is unavailable, the SelfNodeRemediationConfig CR uses a software reboot.
+3. isSoftwareRebootEnabled: true - Specify if you want to enable software reboot of the unhealthy nodes. By default, the value of isSoftwareRebootEnabled is set to true. To disable the software reboot, set the parameter value to false.
+4. apiServerTimeout: 15s - Specify the timeout duration to check connectivity with each API server. When this duration elapses, the Operator starts remediation. The timeout duration must be greater than or equal to 10 milliseconds.
+5. apiCheckInterval: 5s - Specify the frequency to check connectivity with each API server. The timeout duration must be greater than or equal to 1 second.
+6. maxApiErrorThreshold: 3 - Specify a threshold value. After reaching this threshold, the node starts contacting its peers. The threshold value must be greater than or equal to 1 second.
```

Comment:
After reaching this threshold: does remediation start at 3 errors, or above 3 errors? If the latter, could it be "over this threshold"? Otherwise "When reaching this threshold"?

Reply:
Start at 3 errors, I'll update :+1:

Reply:
> my impression is that we could drop "duration" when talking about "timeout".

TBH both sound ok to me, but since this text  is a copy paste  from the official OCP docs (which was put together by folks whose English is far superior to mine :sweat_smile:)  I'd prefer to stick with the original version, unless it's something you feel very strongly about.